packages/vite-plugin-cloudflare/playground/cloudflare-env/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@playground/cloudflare-env",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "vite build --app",
+		"build:custom-mode": "vite build --app -c vite.config.custom-mode.ts",
+		"check:types": "tsc --build",
+		"dev": "vite dev",
+		"dev:custom-mode": "vite dev -c vite.config.custom-mode.ts",
+		"preview": "vite preview",
+		"preview:custom-mode": "vite preview -c vite.config.custom-mode.ts"
+	},
+	"devDependencies": {
+		"@cloudflare/vite-plugin": "workspace:*",
+		"@cloudflare/workers-tsconfig": "workspace:*",
+		"@cloudflare/workers-types": "^4.20241230.0",
+		"typescript": "catalog:vite-plugin",
+		"vite": "catalog:vite-plugin",
```

Comment:
Ah, I see you are using the catalog here so this needs to be consistent with the plugin `package.json`.

Reply:
e985f966dffc6d2585368fe295338634c56ea909